component/api/src/main/java/org/exoplatform/services/organization/auth/OrganizationAuthenticatorImpl.java

```diff
@@ -0,0 +1,453 @@
+/*
```

Comment:
Can you define this module inside component/identity to make it imported at the same time than Organization Service ?
The API Module doesn't hold any implementation and shouldn't  as you can see, it's made of constants, DTO, Interfaces, exceptions ...